.gitignore

```diff
@@ -128,3 +128,5 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+#Custom
+notes.txt
```

Comment:
Even though you added it notes.txt here, it seems that it was added to the pull request anyway. Just deleted it from it